### ISSUE TITLE
Add specifics about tests to "Tests" view (node, variation, which build agent)

### ISSUE THREAD

**Issue by dslomov** [bazelbuild/continuous-integration#119]
Consider: http://ci.bazel.io/blue/organizations/jenkins/Global%2Fbazel-tests/detail/bazel-tests/130/tests
I can see the tests that are failing (great) with specific log snippets (amazing).
What I cannot see is on which architectures the test has failed, and on which machine. It is really hard to get that information

**Comment by damienmg** [comment]
The XMLs are generated by Bazel itself, so this is technically a bazel bug. Another problem is that the [Ant JUnit XML format](http://windyroad.com.au/dl/Open%20Source/JUnit.xsd) is limited but we can still add the `hostname` attribute.

For latest unfortunately we have a bug that prevent generation of XML file when timing out (we now do) and we did not had standard output (we now have).

**Comment by buchgr** [comment]
The XSD allows us to add custom <property /> tags to a testsuite i.e.

```xml
<testsuite>
  <property name="platform" value="macOS" />
</testsuite>
```

So we could use that too.

**Comment by buchgr** [comment]
> For latest unfortunately we have a bug that prevent generation of XML file when timing out (we now do) and we did not had standard output (we now have).

I don't understand this sentence. Do we still have a bug or not?

**Comment by ulfjack** [comment]
There were two issues in Bazel which were fixed, but haven't been released yet (or have they?), so there may not be a bug anymore. We should check at a release that contains both bug fixes.

**Comment by damienmg** [comment]
The two issues I was talking about have been fixed and should be in 0.5.4
but are not in 0.5.3.

**Comment by philwo** [comment]
Obsolete, as we're migrating away from Jenkins.
